controllers/postgres_controller.go

```diff
@@ -180,55 +179,31 @@ func (r *PostgresReconciler) createOrUpdateZalandoPostgresql(ctx context.Context
 		if !errors.IsNotFound(err) {
 			return fmt.Errorf("failed to fetch zalando postgresql: %w", err)
 		}
-		return r.createZalandoPostgresql(ctx, instance)
-	}
-
-	// Update zalando postgresql.
-	mergeFrom := client.MergeFrom(rawZ.DeepCopy())
-	patched := instance.ToPatchedZalandoPostgresql(rawZ)
-	if err := r.Service.Patch(ctx, patched, mergeFrom); err != nil {
-		return fmt.Errorf("failed to update zalando postgresql: %w", err)
-	}
-	log.Info("zalando postgresql updated", "namespace", patched.Namespace, "name", patched.Name)
-
-	return nil
-}
-
-func (r *PostgresReconciler) createZalandoPostgresql(ctx context.Context, instance *pg.Postgres) error {
-	z := instance.ToZalandoPostgres()
-	namespacedName := types.NamespacedName{Namespace: z.Namespace, Name: z.Name}
-	log := r.Log.WithValues("ns/name", namespacedName)
-
-	// Add custom labels (used in combination with the inherited_labels feature of the operator)
-	z.Labels[pg.TenantLabelName] = instance.Spec.Tenant
-	z.Labels[pg.ProjectIDLabelName] = instance.Spec.ProjectID
 
-	// Make sure the namespace exists in the worker-cluster.
-	ns := z.Namespace
```

Comment:
Don't we still need the creation of the namespace?

Reply:
Currently we ensure the namespace in [`OperatorManager`](https://github.com/fi-ts/postgreslet/blob/e736fa7e4a5d36ecd913606126928dfc044101c3/pkg/operatormanager/operatormanager.go#L97).  It would be better if we have this piece of logic shifted to `Reconcile` so that we have a clear overview there.